### DISCUSSION TITLE
convertUsing misnamed

### DISCUSSION THREAD

**Discussion by mikkel-ol** [nartc/mapper#432] [General]
Coming from .NET AutoMapper and used to its jargon, I think the method `convertUsing()` is mistaken for what AutoMapper calls Custom Value Resolvers.

The naming suggests correspondence to Custom Type Converters in AutoMapper, which is a complete custom mapping from source to destination - *not* custom value/property resolving.

**Comment by nartc** [comment]
Hi @mikkel-ol, the `convertUsing` for `forMember()` was taken from here https://docs.automapper.org/en/stable/Value-converters.html
